cartesi-machine.proto

```diff
@@ -396,6 +396,15 @@ message RunResponse {
     bool iflags_h = 3;
     bool iflags_y = 4;
     bool iflags_x = 5;
+    enum InterpreterBreakReason {
```

Comment:
Huh. I didn't even know you could define an enum inside a message. I'd move it out just to make it more similar to the other languages.